software/IOIOLibAndroid/build.gradle

```diff
@@ -1,16 +1,7 @@
 apply plugin: 'com.android.library'
+apply plugin: 'com.github.dcendents.android-maven'
 
-uploadArchives {
-    repositories {
-        mavenDeployer {
-            pom.project {
```

Comment:
I simplified it. 
If you see a benefit somewhere (and at least working and used) go for it !

Reply:
The stuff I see being lost is the description and details given in the past [maven repo](https://mvnrepository.com/artifact/com.github.ytai.ioio). If you look into a single lib, single version, you see the pom info including [ytai's email](https://mvnrepository.com/artifact/com.github.ytai.ioio/IOIOLibAndroidBluetooth/5.07). This can be an initial point of contact, but I can't claim to ever have used such a path. 

I don't see any similar POM info available on jitpack, but maybe the dev process of using jitpack is just so different that its not necessary? If someone is using the jitpack builds, then they are likely getting there from here (github), and if that is the case, then all that info is already available here. 

On the other hand, if someone finds the jitpack builds somewhere else (e.g. some sample code that imports the ioio jitpack lib), then the first thing they'd see is a description-less version tag and nothing about what the library represents. I guess I see a possibility for confusion, but the likelihood of it actually being a problem VERY small, so I agree, this can be neglected unless someone says otherwise.